gnd/src/main/java/com/google/android/gnd/ui/common/BindingAdapters.java

```diff
@@ -50,6 +59,35 @@ public static void bindText(TextInputEditText view, Response response) {
     }
   }
 
+  @BindingAdapter("android:text")
+  public static void bindText(@NonNull TextView view, Date date) {
+    if (date == null) {
+      return;
+    }
+
+    DateFormat dateFormat = getMediumDateFormat(view.getContext());
+    DateFormat timeFormat = getTimeFormat(view.getContext());
+
+    String newText = dateFormat.format(date).concat(" at ").concat(timeFormat.format(date));
```

Comment:
Please continue to use dot as separator for date/time, since using "at" to prevent the date/time from needing to be localized together as a single phrase.

Reply:
Done